apps/backoffice-v2/src/pages/Home/components/OperationalOverviewSection/OperationalOverviewSection.tsx

```
@@ -0,0 +1,308 @@
+              valueKey="count"
+              config={caseByStatusData.reduce((acc, curr) => {
+                const statusMap = {
+                  'Collection In Progress': { color: '#936AF6' },
+                  'Manual Review': { color: '#007AFF' },
+                  Approved: { color: '#4CAF50' },
+                  Revisions: { color: '#FFB74D' },
+                  Rejected: { color: '#F44336' },
+                };
+
+                return {
+                  ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)